cdp/src/sql-template-string.ts

```diff
@@ -0,0 +1,9 @@
+export const SQL = (sqlParts: TemplateStringsArray, ...args: any[]): { text: string; values: any[] } => {
```

Comment:
This is cool! We just have to not forget the SQL before the backticks

Reply:
Good point! Will add some defence against this 😱